tools/custom-loader.mjs

```diff
@@ -5,12 +5,12 @@ export function resolve(specifier, parent, resolve) {
     let result;
     try {
       result = resolve(specifier, parent);
-    } catch(e) {}
-
+    } catch (e) {
+      //
```

Comment:
Feels like we should log something at least at verbose level here?

Reply:
Possibly out-of-scope for this PR? All I did here was make Lint happy by eliminating the empty-block. The no-op has been there for a long time.